src/components/vue-form-builder.vue

```diff
@@ -165,6 +165,8 @@
 
   import BootstrapVue from "bootstrap-vue";
 
+  import '@processmaker/vue-form-elements/dist/vue-form-elements.css';
```

Comment:
If having a separate CSS import is preferred, styles could be inlined. See https://cli.vuejs.org/guide/build-targets.html#library:
> * `dist/myLib.css`: Extracted CSS file (can be forced into inlined by setting `css: { extract: false }` in `vue.config.js`)